graphdatascience/query_runner/neo4j_query_runner.py

```diff
@@ -218,7 +257,11 @@ def call_procedure(
         query = f"CALL {endpoint}({params.placeholder_str()}){yields_clause}"
 
         def run_cypher_query() -> DataFrame:
-            return self.run_cypher(query, params, database, custom_error)
+            if retryable:
+                routing = neo4j.RoutingControl.WRITE if "write" in endpoint else neo4j.RoutingControl.READ
+                return self.run_retryable_cypher(query, params, database, custom_error, routing=routing)
```

Comment:
Will this ever be a safe combination. Our write are not always idempotent.

Reply:
the main case was `gds.arrow.write.v3`

Maybe should rather have it explicit in for the write protocol case

Reply:
okay its already explicit as before the caller needs to have passed `retryable=True` explicitly

Reply:
Maybe instead of doing the string matching we could pass in the RoutingControl from the caller